src/lib.rs

```diff
@@ -31,6 +41,9 @@ mod socket_listener;
 #[cfg(any(unix, feature = "dox"))]
 mod unix_socket_address;
 mod file;
+mod pollable_input_stream;
+mod pollable_output_stream;
+mod file;
```

Comment:
Seems this effect of many rebases: second `mod file`

Reply:
Seem, commit `Implement File::replace_contents_async manually` was not fully removed from this PR.

Reply:
Fixed both, thanks